src/js/mediaStore.js

```diff
@@ -0,0 +1,108 @@
+import { MediaUIEvents } from './constants.js';
+
+// Very similar to "MediaUIStates" except:
+export const stateFacade = {
+  mediaPaused: {
+    // renaming to "stateOwners" to indicate it is generic
```

Comment:
This terminology threw me off at first but I think I get it. It's not the final state value at this point, it's essentially the state ingredients or sources. stateOwner might not be the most accurate description because of that relationship but I'll sit with it.

I can see how it can make sense to call fullscreenElement a stateOwner, and update that in the store whenever the reference changes in the same way the media element is handled, but what about other controller settings like default-stream-type? What would be the owner of that?

Reply:
an alternative approach would be to just treat these as part of the `state` itself. I can see reasons to treat them as special, but there are some benefits of allowing folks to consume the media element itself as state (say, for advanced use cases or "not yet modeled in state" use cases.